server/@types/forms/index.d.ts

```diff
@@ -56,6 +56,18 @@ declare module 'forms' {
     title: string
   }
 
+  export interface CompleteGoalForm {
+    reference: string
+    title: string
+    note: string
+  }
+
+  export interface CompleteOrArchiveGoalForm {
```

Comment:
We _might_ be able to do this without a form (cant remember what the user journey/screen looks like), but using a form as you have done is fine 👍 . Perhaps we can look at doing it without a form submission some other time 👍

Reply:
yeah I did wonder about this - the form is purely to 'carry' the goal reference during the navigation.